src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+
+    private String name = "Square";
+    private double firstLeg, secondLeg;
+    private String color;
+
+    public Square(double firstLeg, String color) {
+        this.firstLeg = firstLeg;
+        secondLeg = firstLeg;
+        this.color = color;
+    }
+
+    public Square() {
+    }
+
+    @Override
+    public double findArea() {
+        return firstLeg * secondLeg;
```

Comment:
Since the square has all sides equal, the area can be calculated as 'firstLeg * firstLeg' or using a single 'side' variable.